huaweicloud/resource_huaweicloud_gaussdb_opengauss_instance.go

```diff
@@ -686,5 +684,25 @@ func resourceOpenGaussInstanceUpdate(d *schema.ResourceData, meta interface{}) e
 	}
 	log.Printf("[DEBUG] Successfully updated instance %s", instanceId)
 
+	if d.HasChange("backup_strategy") {
+		var updateOpts backups.UpdateOpts
+		backupRaw := d.Get("backup_strategy").([]interface{})
+		rawMap := backupRaw[0].(map[string]interface{})
+		keep_days := rawMap["keep_days"].(int)
+		updateOpts.KeepDays = &keep_days
+		updateOpts.StartTime = rawMap["start_time"].(string)
+		// Fixed to "1,2,3,4,5,6,7"
+		updateOpts.Period = "1,2,3,4,5,6,7"
+		// Fixed to "30"
+		updateOpts.DifferentialPeriod = "30"
```

Comment:
we can use a more graceful way, like:
updateOpts := backups.UpdateOpts {
   ....
}

Reply:
fixed